src/AtomShell/install.jl

```diff
@@ -59,12 +63,17 @@ function install()
            Julia, so instead we look in the default Julia binary location and
            pick the latest version.
            =#
+           
            juliafolders = filter(readdir(ENV["LOCALAPPDATA"])) do f
                                      startswith(f, "Julia-")
               end
            juliaversions = VersionNumber.([replace(f, "Julia-" => "") for f in juliafolders])
            i = findlast(isequal(maximum(juliaversions)), juliaversions)
            zipper = joinpath(ENV["LOCALAPPDATA"], juliafolders[i], "bin", "7z.exe")
+           if !isfile(zipper)
```

Comment:
I think you can remove this now that you've made the change above.

Reply:
Are you sure? It is still possible that in the manual search we find a version of Julia which is 1.3.0, which then triggers this piece of code.